main/common/windows.js

```diff
@@ -0,0 +1,102 @@
+const {Menu, MenuItem, nativeImage} = require('electron');
```

Comment:
Use `'use strict';` at the top of all JS files in the main process, since they're not transpiled, so they're not automatic strict.